keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -937,9 +952,6 @@ The entire scheduling section is tentative. Key opens:
 - Support arbitrary combinations of user- vs. Kubernetes-managed ResourceClaims
   and immediate vs. late allocation?
   https://github.com/kubernetes/enhancements/pull/3064#discussion_r901948474
-- Can and should `SelectedNode`, `SelectedUser`, `Deallocate` be moved to
-  `PodScheduling` or be handled differently?
-  https://github.com/pohly/enhancements/pull/13/files
```

Comment:
SelectedNode can be moved. SelectedUser becomes obsolete (because we are doing pod scheduling, the pod is the intended user by definition). Deallocate must stay where it is to ensure atomic status updates.